### DISCUSSION TITLE
Prometheus metrics per node

### DISCUSSION THREAD

**Discussion by heha1012** [emqx/emqx#6903] [Q&A]
Hi there,
we use EMQX 4.3.11 in a cluster with several nodes.
We also have prometheus plugin installed and scrape these metrics for further monitoring with Grafana.

Some metrics like emqx_messages_publish send these informations **per node** in irregular intervals. 
Is there an indentifier to recognize which node send the respective metric?

![Screenshot 2022-01-31 141412](https://user-images.githubusercontent.com/38078413/151800002-b17488eb-6727-4bfa-9548-c388b906b042.jpg)

**Comment by k32** [comment]
All metrics in prometheus are tagged by the host, so you should be able to add node to the query, as well as aggregate the metric for the entire cluster.
